Aliucord/src/main/java/com/aliucord/api/CommandsAPI.java

```diff
@@ -51,21 +51,52 @@
         public String username;
         public String avatarUrl;
 
+        /**
+         * calls {@link CommandResult#CommandResult(String, List, boolean)} with default arguments.
+         * @see CommandResult#CommandResult(String, List, boolean)
+         */
         public CommandResult() { this(null); }
+
+        /**
+         * calls {@link CommandResult#CommandResult(String, List, boolean)} with default arguments.
+         * @param content Output message content
+         * @see CommandResult#CommandResult(String, List, boolean)
+         */
         public CommandResult(String content) {
             this(content, null, true);
         }
+
+        /**
+         * @param content Output message content
+         * @param embeds Embeds to include in the command output. Requires <code>send</code> to be false.
+         * @param send Whether to send the message or not. If false, messages will appear locally, otherwise they'll be sent to the current channel.
+         */
         public CommandResult(String content, List<MessageEmbed> embeds, boolean send) {
             this.content = content;
             this.embeds = embeds;
             this.send = send;
         }
+
+        /**
+         * @param content Output message content
+         * @param embeds Embeds to include in the command output. Requires <code>send</code> to be false.
+         * @param send Whether to send the message or not. If false, messages will appear locally, otherwise they'll be sent to the current channel.
+         * @param username Username for Clyde's customization. Requires <code>send</code> to be false.
+         */
         public CommandResult(String content, List<MessageEmbed> embeds, boolean send, String username) {
             this.content = content;
             this.embeds = embeds;
             this.username = username;
             this.send = send;
         }
+
+        /**
+         * @param content Output message content
+         * @param embeds Embeds to include in the command output. Requires <code>send</code> to be false.
+         * @param send Whether to send the message or not. If false, messages will appear locally, otherwise they'll be sent to the current channel.
+         * @param username Username for Clyde. Requires <code>send</code> to be false.
+         * @param avatarUrl Avatar URL for Clyde, must be raw. Requires <code>send</code> to be false.
```

Comment:
what does 'must be raw' mean

Reply:
Should be an actual path to the file and not a redirect, as redirects don't seem to work

Reply:
so it cant be a proxy url? raw makes me think of github raw links so its kinda confusing xD maybe change it to "must be a direct link, not a redirect"

Reply:
Yep, sounds better